### ISSUE TITLE
Regression - NHL 15

### ISSUE THREAD

**Issue by Kravickas** [RPCS3/rpcs3#3156]
Hi,
first and last build that was rendering game is [0.0.3-5541 5b61dcdc](https://ci.appveyor.com/project/rpcs3/rpcs3/build/0.0.3-5541/artifacts)    
Build before and after showing just white/blue-ish screen.
Master build that broke it [0.0.3-5543 b3968261](https://ci.appveyor.com/project/rpcs3/rpcs3/build/0.0.3-5543/artifacts)

Newest build [0.0.3-5575 874a0a3b](0.0.3-5575) - [RPCS3.zip](https://github.com/RPCS3/rpcs3/files/1202563/RPCS3.zip) - white screen
First and last working build - [RPCS3ok.zip](https://github.com/RPCS3/rpcs3/files/1202564/RPCS3ok.zip) - rendering game

Same settings just copied old working build min. later after testing newest build with white screen and started rendering  game again on old build.

My post about it with screenshots- [forum post](http://www.emunewz.net/forum/showthread.php?tid=193318&pid=281583#pid281583)

**Comment by Kravickas** [comment]
I think problem is in #3096

that he deleted 

```
-		/*
-
-		// Store previous fbo addresses to detect RTT config changes.
-		std::array<u32, 4> m_previous_color_address = {};
-		u32 m_previous_address_z = 0;
-		u32 m_previous_target = 0;
-		u32 m_previous_clip_horizontal = 0;
-		u32 m_previous_clip_vertical = 0;
-		*/
```
yes there was error E {rsx::thread} RSX: FBO check failed: 0x8cd6, but it was rendering game.

**Comment by Kravickas** [comment]
Fixed by #3458
